Documentation/README_InteractiveElement.md

```diff
@@ -0,0 +1,91 @@
+# Interactive Element 
+
+An Interactive Element is the next generation of MRTK's Interactable. An Interactive Element is an object that receives input and fires events depending on the type of input.  The type of input supported is defined in the form of states. 
+
+## Adding a Core State
+
+A CoreInteractionState is a state where the setting logic is already defined. For example, in BaseInteractiveElement, the Focus state is set to on when the IMixedRealityFocusHandler's OnFocusOn is called, making the state setting logic already defined.
+
+Currently, Focus is the only core state supported but the following core states will be added in the future:
+
+![AddCoreState](Images/InteractionElements/HowToExamples/NewCoreStates.png)
+
+All Core Interaction States will have an associated custom event configuration.
```

Comment:
What is a custom event configuration? Should this maybe link to the definition? I haven't seen that defined before.

Reply:
Each state has an event configuration.  For example the Focus state has the `FocusEvents` event configuration.  

A state's event configuration is a scriptable object that contains the serialized Unity Events with specific event data for a state. For example, the `FocusEvents` event config contains serialized OnFocusOn and OnFocusOff Unity Events with Focus Event Data. The event configuration's purpose is for event exposure in the inspector and to serve as the entry point to add listeners to a state event via script.